locations/spiders/currys.py

```diff
@@ -1,60 +1,39 @@
 # -*- coding: utf-8 -*-
-import scrapy
-from locations.items import GeojsonPointItem
-
-countries = ["England", "Uk", "Ireland", "Scotland"]
+import json
 
-formdata = {
-    "sFormName": "storeFinder",
-    "storeFind": "true",
-    "subaction": "stores_map",
-    "sStoreKeywordHidden": "england",  # Previous search. The choice of england was arbitrary
-    "sStoreGeo": "",  # Appears to not be used
-    "sStoreKeyword": "",
-}  # Search term
-
-headers = {
-    "Referer": "https://www.pcworld.co.uk/gbuk/s/find-a-store.html",
-    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64…) Gecko/20100101 Firefox/57.0",
-}
+from locations.hours import OpeningHours
+from locations.items import GeojsonPointItem
+from scrapy.spiders import SitemapSpider
 
 
-class CurrysSpider(scrapy.Spider):
+class CurrysSpider(SitemapSpider):
     name = "currys"
     item_attributes = {"brand": "currys", "brand_wikidata": "Q3246464"}
-    allowed_domains = ["https://www.pcworld.co.uk"]
-
-    def start_requests(self):
-        search_url = "https://www.pcworld.co.uk/gbuk/s/find-a-store.html"
-        for country in countries:
-            formdata["sStoreKeyword"] = country
-            request = scrapy.FormRequest(
-                search_url, formdata=formdata, callback=self.parse
-            )
-            request.meta["country"] = country
-            yield request
+    allowed_domains = ["www.currys.co.uk"]
+    sitemap_urls = ["https://www.currys.co.uk/sitemap-stores.xml"]
```

Comment:
Not sure where you're at with the Currys bot, but the URL for their stores XML file is https://www.currys.co.uk/sitemap-stores-curryspcworlduk.xml as opposed to the one listed in this PR - hope that helps.

Reply:
Thanks, I've updated this, but the issue here is with @cloudflare they are blocking the traffic. You can test with: `$ curl https://www.currys.co.uk/sitemap-stores-curryspcworlduk.xml`